broker/src/main/java/org/apache/rocketmq/broker/processor/QueryAssignmentProcessor.java

```diff
@@ -220,13 +220,13 @@ private RemotingCommand queryAssignment(ChannelHandlerContext ctx, RemotingComma
                         } else {
                             if (cidAll.size() <= mqAll.size()) {
                                 //consumer working in pop mode could share the MessageQueues assigned to the N (N = popWorkGroupSize) consumer following it in the cid list
-                                allocateResult = allocateMessageQueueStrategy.allocate(consumerGroup, clientId, mqAll, cidAll);
+                                allocateResult = allocateMessageQueueStrategy.allocate(null, consumerGroup, clientId, mqAll, cidAll);
```

Comment:
Is it compatible with the previous version?

Reply:
QueryAssignmentProcessor and AllocateMessageQueueStrategy always  has same version, new version of QueryAssignmentProcessor will not depend on old AllocateMessageQueueStrategy interface.